README.md

```diff
@@ -48,10 +57,13 @@ We believe the W3C's purpose is to:
 * Establish and improve collaborative relationships with other organizations in the domain of Internet and Web standards.
 
 # Identity
-The global brand of the W3C should be centered around the following:
-* We are an open forum where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies.
-* We are socially responsible and committed to ensure that the Web is for everyone; we greatly emphasize accessibility, internationalization, privacy, security and diversity.
-* We are committed to developing open and royalty-free standards with high focus on interoperability and collective empowerment.
+
+We are:
+* an open forum where diverse voices from around the world and industries come together, incubate and build consensus for global standards for Web technologies.
+* socially responsible and committed to ensure that the Web is for everyone; we greatly emphasize accessibility, internationalization, privacy, security and diversity.
+* committed to developing open and royalty-free standards with high focus on interoperability and collective empowerment.
+
+These define the global brand of the W3C.
```

Comment:
I'm not sure this sentence is even necessary?

Reply:
It was the lead-in to the three bullet points, so I was hesitant to delete it.

Reply:
Meh.  I think "Identity" is clear enough. I'm okay either way.